catalog/spatial/registry/registry-rest-endpoint/src/test/resources/csw-simplified-registry-package.xml

```diff
@@ -33,6 +33,14 @@
     <rim:RegistryObjectList>
         <!-- defines the general node information -->
         <rim:ExtrinsicObject id="urn:registry:node" objectType="urn:registry:federation:node">
+            <rim:Description>
+                <rim:LocalizedString
+                        value="Extrinsic Object Description"/>
+            </rim:Description>
+            <rim:Name>
+                <rim:LocalizedString value="Extrinsic Object Name"/>
+            </rim:Name>
```

Comment:
No, it's just a sample version in the sample xml. Has nothing to do with the version of DDF.